pod/perlvar.pod

```diff
@@ -1872,6 +1872,10 @@ It has the same scoping as the C<$^H> and C<%^H> variables.  The exact
 values are considered internal to the L<warnings> pragma and may change
 between versions of Perl.
 
+Each time a statement completes being compiled, the current value of
+C<${^WARNING_BITS}> is stored with that statement, and can later be
+retrieved via C<(caller($level))[9]>.
+
```

Comment:
Are these three documentation changes (here + the other two lower down) intended to be being added? They don't appear to relate to the current change. Perhaps it's some unrelated collateral from the double-revert?

Reply:
Yes they are related. See thread starting at:

   https://www.nntp.perl.org/group/perl.perl5.porters/2020/02/msg257128.html

Sayeth Dave Mitchell in 

   https://www.nntp.perl.org/group/perl.perl5.porters/2020/03/msg257214.html
 
The change in behaviour was a side-effect of my fix for recording caller
data correctly, but I think the change makes behaviour actually match the
docs (although they are a bit vague). So I think the bug-fix is correct.